gsonvalue-processor/src/main/java/me/tatarka/gsonvalue/GsonValueProcessor.java

```diff
@@ -207,8 +207,13 @@ private void process(Element element) throws IOException {
 
         // @Override public T read(JsonReader in) throws IOException
```

Comment:
`write` should be handled as well, since it can be called when you use `.serializeNulls()`.

Reply:
I was planning on a separate pull request for the `write` case, but I can add that in this pull request later if you prefer.